util/pastPerformance/testReleasesPerformance

```diff
@@ -151,6 +151,10 @@ export CHPL_TEST_PERF_DATE="10/08/20"
 export CHPL_HOME=$chpl_release_home/chapel-1.23.0
 testReleasePerformance
 
+export CHPL_TEST_PERF_DATE="3/18/21"
```

Comment:
I think that this date should match the branch date (both intuitively, and pattern-matching against the previous entries).

Reply:
Correct, that date should be the date of the branch, not the release.